algorithms/offline/cql.py

```diff
@@ -232,7 +233,7 @@ def modify_reward(dataset, env_name, max_episode_steps=1000):
         dataset["rewards"] /= max_ret - min_ret
         dataset["rewards"] *= max_episode_steps
     elif "antmaze" in env_name:
-        dataset["rewards"] -= 1.0
+        dataset["rewards"] = dataset["rewards"] * 10.0 - 5.0
```

Comment:
Should we do something like:
```python
dataset["rewards"] * scale + bias
```
here? Taken from https://github.com/tinkoff-ai/CORL/pull/52/files

Reply:
Changed it. Also added it to the offline-to-online CQL and Cal-QL